Establish FAQ section pattern and add FHIR Developers FAQ

MaxPhilips opened the pull request:
Did this in a way that matches what is set up for the may2015 and dstu1 sections of the site.  We will be able to add new FAQ sections by adding new faq/*.md files and adding links in faq_sidebar.html

I added links to the FAQ in the header and in the 'highlights' section.

MaxPhilips commented on layouts/highlights.html:
```diff
@@ -12,6 +12,11 @@
         </li>
         <li class="highlight-module">
+            <a href="/faq/"><span class="mega-octicon octicon-question"></span></a>
+            <h2><a href="/faq/">FHIR Developer FAQ</a></h2>
```
Probably want this title to just be "FAQ" since it could eventually link to an FAQ page with other FAQ sections besides FHIR Developer FAQs

MaxPhilips replied:
Updated to be more generic: https://github.com/cerner/fhir.cerner.com/commit/1c26adb2c23ad42941ea18c686933277fa94e709#diff-dc155e29ed33ac86bdcb09416efdef38L16

jfrhong commented:
+1

icon5585 commented:
+1

daliboz commented:
I don't think we need the intermediary "Overview" on the FAQ page. Just having the FAQs be the only section would be fine.

MaxPhilips commented:
Removed Overview from content/faq.md and moved the content from content/faq/developer.md to content/faq.md with 1c26adb

Also updated the links in layouts/faq_sidebar.html

daliboz commented:
Posted a question out to others on rewording of the first item in the FAQ. Otherwise, +1

MaxPhilips commented:
Added 7b382c3 with updated wording.

MaxPhilips commented:
Squashed commits, merging
